<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogApiTest</class>
 <widget class="QWidget" name="DialogApiTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>356</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Application</string>
  </property>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>303</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExec">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>303</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exec</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkBoxRealtime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Realtime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSource0">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="OperationsGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Operations</string>
   </property>
   <widget class="QRadioButton" name="radioButtoncopyMakeBorder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>copyMakeBorder</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCodeScanner_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CodeScanner</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtoncalcBackProject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>calcBackProject</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSource1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
